README.md

```diff
@@ -203,6 +203,8 @@ You can follow the log file by running the following:
 docker logs -f irods_s3_api
 ```
 
+This application can also run against iRODS 4.2.11 and 4.2.12. No adjustments to your configuration file are required.
```

Comment:
Is Docker the only way to run this at the moment? If not, might be worth mentioning that the application cannot be installed as a package alongside a 4.2.11/4.2.12 server on the same host.

Reply:
We're only targeting Docker for now.    Yes, we should probably have a sentence about not being on the same machine as 4.2.x.

Reply:
Added more words after the highlighted line.